src/routes/forgot-password/components/ForgotPasswordPage.js

```
@@ -0,0 +1,86 @@
+const { useState } = require('react');
+const T = require('prop-types');
+const { default: Styled } = require('styled-components');
+const { default: Typography } = require('@mui/material/Typography');
+const { default: TextField } = require('@mui/material/TextField');
+const { default: Button } = require('@mui/material/Button');
+const { default: Box } = require('@mui/material/Box');
+
+const internals = {};
+
+module.exports = function ForgotPasswordPage({ onPressForgotPassword }) {
+
+    const [email, setEmail] = useState('');
+    const [isSubmitting, setIsSubmitting] = useState(false);
+    const [didEmailSend, setDidEmailSend] = useState(false);
+
+    const disableSubmit = () => {
+
+        return !email;
+    };
+
+    const handleSubmit = async (ev) => {
+
+        ev.preventDefault();
+        const accountInfo = formatFields();
+        setIsSubmitting(true);
+        const [err] = await onPressForgotPassword(accountInfo);
+        setIsSubmitting(false);
+        if (!err) {
```

should we surface this error to the user somehow?  I noticed login and register also don't handle errors, so maybe we could split this off and tackle them all at once.  Could be a good time to bring in snackbar functionality as well.

---

This is a great idea—we already had a card in our development board to handle snackbar integration and some other auth improvements (which I apparently personally made in 2019 😅). The good news is we're doing a good amount of that work here, so I'm repurposing it to cover snackbar integration and error handling :+1:

---

This is handled in #288, marking resolved